### PR TITLE
Google analytics on forgotPassword. Set username focus. Remove style inline.

### PR DESCRIPTION
**Pull request by absis**

### CODE REVIEW

File: src/browserNotSupported.html [hunk 1]
```diff
@@ -15,20 +15,32 @@
...
+        <div id="cloud-2"></div>
+        <div id="cloud-3"></div>
+        <div id="cloud-4"></div>
+    </div>
     <div id="loginGeneralContainerCentered" class="login-text">
         <div id="loginGeneralContainer" class="login-text">
             <div id="logo_eyeos"></div>
             <div id="formBox" class="formBox">
-                <p id="notSupported" class="almost_there">
+                <p class="wellcome_title almost_there">
```
Comment (jordins) [inline src/browserNotSupported.html]: typo!

File: src/browserNotSupported.html [hunk 2]
```diff
@@ -37,14 +49,21 @@
...
-
+            <div id="desktop_version_container">
+                <a id="desktopVersionLink" href="#">Continue to the desktop version</a>
+            </div>
         </div>
     </div>
 </div>
 <script>
-    window.currentPage = "browserNotSupported";
+    window.currentPage = "browserNotSupported"
```
Comment (jordins) [inline src/browserNotSupported.html]: forgotten semicolon

File: src/browserNotSupported.html [hunk 3]
```diff
@@ -37,14 +49,21 @@
...
 </script>
 <script src="vendor/require.js" data-main="js/main.js"></script>
+<script>
+    $("#desktopVersionLink").click(function () {
```
Comment (jordins) [inline src/browserNotSupported.html]: Can you move it to a script file?

File: src/browserNotSupported.html (hunk 2 above)
Comment (absis) [inline src/browserNotSupported.html]: Done

File: src/browserNotSupported.html (hunk 3 above)
Comment (absis) [inline src/browserNotSupported.html]: Done

File: src/browserNotSupported.html (hunk 1 above)
Comment (absis) [inline src/browserNotSupported.html]: Done

**Pull request merged**